### ISSUE TITLE
FeedbackSessionsLogicTest: use a cleaner method to compare the csv results

### ISSUE THREAD

**Issue by wkurniawan07** [TEAMMATES/repo#4629]
The current method is:
```java
assertEquals(exportLines[0], "some line");
assertEquals(exportLines[1], "some other line");
// ... repeat as needed
```
which can be converted to:
```java
String[] expected = {
   "some line",
   "some other line",
   // repeat as needed
   "the last line"
}
assertArrayEquals(expected, exportLines); // this is a built-in JUnit method
```
Advantages:

1. The expected and actual are at the correct parameter positions.
2. There's an extra check on the array size (previously, who knows if we miss some of the `exportLines`?).
3. The way the `expected` is presented actually plays a double role to show how the export should look like as well (only differs in `" -> \"` and maybe some variables), so we might not need the comments entirely for `This is how the export should look like...` anymore. This should be a good thing: why do we need to change _two_ items when the CSV changes?

**Comment by wkurniawan07** [comment]
Gave a quick try to one and guess what, it fails right away. So we've been including incomplete tests all these while.

**Comment by LowWeiLin** [comment]
Can we simply do a string comparison? There should not be a need to split the csv into lines other than for readability in the tests, which can be done by proper formatting.
```
String expected = "some line" + Const.EOL +
    "some other line" + Const.EOL +
    // repeat as needed
    "the last line";

assertEquals(expected, exportLines);
```

**Comment by wkurniawan07** [comment]
I think that's even better, would solve #4630 concurrently as well. We can even take it one step further like this:
```java
String[] expected = {
    ...
}
assertEquals(StringUtils.join(expected, Const.EOL), export);
```
to suppress the repetition of `Const.EOL`.

**Comment by jkt001** [comment]
I might look into this issue and solve it over the weekend. I'll let you know of the progress by next Monday.

**Comment by wkurniawan07** [comment]
Your call. You can mention fixing both issues on the PR because the solution does just that.

**Comment by jkt001** [comment]
I have tried a quick example (in the part that is causing #4630 as well) and it managed to pass the test-case on my Mac, which uses <code>\n</code> as the newline character.

As for the incomplete tests that @wkurniawan07 mentioned, it seems that a few blank lines exist after the last line (at least for the part that I have fixed) and the previous test did not account for these blank lines. I will look into the entire test file and see if there are any other incomplete tests.

I will work on this entire file and submit a PR once it is done. The PR will fix this and #4630 concurrently because of the nature of the fix.
